test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -197,6 +197,7 @@ AndroidArgs
     flank:
       testShards: 7
       repeatTests: 8
+      junitGcsPath:${' '}
```

Comment:
What's the `${' '}`?

Reply:
That's an empty space. Regular empty spaces are auto trimmed by IntelliJ.